cpg_utils/hail_batch.py

```diff
@@ -81,11 +54,21 @@ def copy_common_env(job: hb.batch.job.Job) -> None:
     passed through for "batch-in-batch" use cases.
 
     The environment variable values are extracted from the current process and
-    copied to the environment dictionary of the given Hail Batch job.
-    """
-    # If possible, please don't add new environment variables here, but instead add
-    # config variables.
-    for key in ('CPG_CONFIG_PATH',):
+    copied to the environment dictionary of the given Hail Batch job."""
+
+    for key in (
+        'CPG_ACCESS_LEVEL',
```

Comment:
Looks like there's still some merge remnants.

Reply:
Somehow screwed up merge with main. Should be good now!